docs/openapi/components/schemas/credentials/PurchaseOrderCredential.yml

```diff
@@ -49,7 +49,101 @@ properties:
   expirationDate:
     type: string
   issuer:
-    $ref: ../snippets/IssuerOrganization.yml
+    title: Issuer Organization
```

Comment:
We really should just have a common snippet for issuer org and use that. Instead of the way we've been in lining it

Reply:
Cheers, @mkhraisha!

We're consciously moving away from `$ref: IssuerOrganization`: 

1. References in general makes schemas harder to consume and should be avoided. 
2. The kind of issuer organization isn't always the same and benefits from targeted descriptions and fields. Buyer issues PO, Seller issues CI, Shipper issues SI, Carrier issues BL, etc...